requirements.txt

```diff
@@ -1,3 +1,3 @@
-requests>=2.7.0
-Pillow==4.0.0
-six>=1.10.0
+requests==2.7.0
+Pillow==6.2.2
```

Comment:
Lock to specific versions until we've migrated to python 3. Bump Pillow to remediate CVEs